src/main/java/com/ibm/stocator/fs/swift/SwiftAPIDirect.java

```diff
@@ -86,6 +88,24 @@ public static SwiftInputStreamWrapper getObject(Path path, JossAccount account,
   }
 
   /**
+   *
+   */
+  public static HttpResponse getObjectMetadata(JossAccount account, String container, String object,
+                                               SwiftConnectionManager scm) {
+    HttpHead headRequest = new HttpHead(account.getAccessURL() + "/" + container + "/" + object);
+    headRequest.addHeader("X-Auth-Token", account.getAuthToken());
+    HttpClient client = scm.createHttpConnection();
+    HttpResponse metadata = null;
+    try {
+      metadata = client.execute(headRequest);
+    } catch (IOException e) {
+      LOG.error("Get object metadata operation could not be completed.");
+      e.printStackTrace();
+    }
+    return metadata;
+  }
+
+  /**
```

Comment:
The code for success is still 200 in this case. What should happen if the code isn't 200? Some of the logic depends on this method returning a null.